packages/components/src/NotificationNext/NotificationContainerNext/styles.ts

```diff
@@ -113,9 +134,20 @@ export const Wrapper = styled.div`
       height: 3px;
     }
   }
+
+  @media only screen and (width <= 480px) {
```

Comment:
```suggestion
  ${({ theme }) => theme.breakpoints.down('sm')} {
```

медиа-запрос должен быть из темы